src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1565,6 +1565,7 @@ public static class UI
          public static readonly string ScaleButton = "Scale";
          public static readonly string AllowsScientificNotation = "Allows scientific notation";
          public static readonly string ShouldRestoreWorkspaceLayout = "Restore opened view when loading project";
+         public static readonly string ShouldColorGroupObservedData = "Color group observed data from same folder when dropping to chart";
```

Comment:
You can define this in core. It should be the same for PKSim and MoBi right?

Reply:
done